toolbox

```diff
@@ -964,7 +964,12 @@ remove_containers()
             ret_val=$(echo "$ids" \
                       | (
                             while read -r id; do
-                                if ! $prefix_sudo podman rm $force_option "$id" >/dev/null 2>&3; then
+                                $prefix_sudo podman rm $force_option "$id" >/dev/null 2>&3
+                                ret_code=$?
+                                if [ $ret_code = 2 ]; then
+                                    echo "$base_toolbox_command: cannot remove running or paused container $id" >&2
```

Comment:
As part of #59, we have been trying to make the commands somewhat self-documenting. In this case, it might be nice to suggest the `--force` option and/or `podman stop`. See some of the recent changes done in #59 for ideas.